contrib/Joshua/scripts/localClusterStart.sh

```diff
@@ -161,27 +175,60 @@ function createClusterFile {
 	return ${status}
 }
 
+# Stop the Cluster from running.
+function stopCluster {
+	# Add an audit entree, if enabled
+	if [ "${AUDITCLUSTER}" -gt 0 ]; then
+		printf '%-15s (%6s)  Stopping cluster %-20s (%6s): %s\n' "$(date +'%Y-%m-%d %H:%M:%S')" "${$}" "${CLUSTERSTRING}" "${FDBSERVERID}" >> "${AUDITLOG}"
+	fi
+	if [ -z "${FDBSERVERID}" ]; then
+		log 'FDB Server process is not defined'
+		let status="${status} + 1"
+	elif ! kill -0 "${FDBSERVERID}"; then
+		log "Failed to locate FDB Server process (${FDBSERVERID})"
+		let status="${status} + 1"
+	elif "${BINDIR}/fdbcli" -C "${FDBCONF}" --exec "kill; kill ${CLUSTERSTRING}; sleep 3" --timeout 120 &>> "${LOGDIR}/fdbcli-kill.log"
+	then
+		log "Killed cluster (${FDBSERVERID}) via cli"
```

Comment:
Should we add a check to confirm that the process is dead?

Reply:
done